### ISSUE TITLE
Update UI icons

### ISSUE THREAD

**Issue by javerbukh** [glue-viz/glue-jupyter#192]
We have UI changes planned in jdaviz and are wondering if these changes can also be applied upstream to glue-jupyter.

https://github.com/spacetelescope/jdaviz/issues/138

**Comment by astrofrog** [comment]
Yes provided they are generic enough, but note that icons actually live in glue-core at the moment and are used in Qt. Could the icons be used in the Qt app too?
